sdk/servicebus/azure-servicebus/azure/servicebus/management/__init__.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._management_client import ServiceBusManagementClient
+from ._shared_key_policy import ServiceBusSharedKeyCredentialPolicy
+from ._generated.models import QueueDescription, AuthorizationRule, MessageCountDetails, QueueRuntimeInfo, \
+    AccessRights, EntityAvailabilityStatus, EntityStatus
+
+__all__ = [
+    "ServiceBusManagementClient",
+    "ServiceBusSharedKeyCredentialPolicy",
```

Comment:
Doesn't the AMQP-based client used Shared Key as well?
Why do we have a different policy here?

Reply:
It looks like this is an actual pipeline policy object - we don't tend to expose those, rather we take a ShareKeyCredential object of some kind, and turn it into a policy internally.

Reply:
Yes this is a pipeline policy that uses shared key credential. No problem to hide it.
But if we allow users to create their own pipeline and pass the pipeline to ServiceBusManagementClient, they might need this policy object?

Reply:
Removed it anyway. Will discuss whether to expose it later.